src/brokers/metrics/components/CardBrokerCPUUsageMetrics/CardBrokerCPUUsageMetrics.container.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC, useEffect, useState } from 'react';
+import { CardBrokerCPUUsageMetrics } from './CardBrokerCPUUsageMetrics';
+import {
+  usePrometheusPoll,
+  PrometheusEndpoint,
+  PrometheusResponse,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { parsePrometheusDuration } from '../../../../utils';
+import { getMaxSamplesForSpan } from '../../utils';
+
+export type CardBrokerCPUUsageMetricsContainerProps = {
+  name: string;
+  namespace: string;
+  defaultSamples?: number;
+  timespan?: number;
+  fixedEndTime?: number;
+};
+
+type AxisDomain = [number, number];
+
+export const CardBrokerCPUUsageMetricsContainer: FC<
+  CardBrokerCPUUsageMetricsContainerProps
+> = ({ name, namespace = 'deafult', defaultSamples = 300, timespan }) => {
```

Comment:
Not sure if having the 'deafult' instead of 'default' makes any difference or not.